spec/presenters/two_factor_options_presenter_spec.rb

```diff
@@ -87,5 +87,17 @@
         )
       end
     end
+
+    context 'when priority is usability ordered in a/b testing bucket' do
+      let(:presenter) do
+        described_class.new(user_agent: user_agent, priority_bucket: :usability_priority)
+      end
+
+      it 'supplies auth app as the first option' do
```

Comment:
Description here appears to be inaccurate, maybe missed in copy from other spec.

```suggestion
      it 'supplies phone as the first option' do
```